src/mscorlib/corefx/System/Globalization/Calendar.cs

```diff
@@ -840,7 +840,6 @@ internal static long TimeToTicks(int hour, int minute, int second, int milliseco
             throw new ArgumentOutOfRangeException(null, SR.ArgumentOutOfRange_BadHourMinuteSecond);
         }
 
-        [System.Security.SecuritySafeCritical]  // auto-generated
```

Comment:
Yes I understand this. my question was regarding if we are going to resurrect back System.Security.SecuritySafeCritic type (as we are bringing back a lot of other types), do we need to honor these attributes if we brought back this type? 

cc @weshaggard